src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadPropertiesStepExecution.java

```diff
@@ -64,7 +64,9 @@
         if (!StringUtils.isBlank(step.getFile())) {
             FilePath f = ws.child(step.getFile());
             if (f.exists() && !f.isDirectory()) {
-                properties.load(f.read());
+                try(InputStream is = f.read()){
+                   properties.load(is); 
```

Comment:
oh, then I guess it's ok :) I've got too many plugins with various requirements on java level, it seems I'm mixing them up :)